bin/deploy.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# This file is used to deploy Test Pilot to an S3 bucket.  It expects to be run
+# from the root of the Test Pilot directory and you'll need your S3 bucket name
+# in an environment variable $BUCKET
+
+# Questions?  Hit up #testpilot on IRC
+
+if [ ! -d "dist" ]; then
+    echo "Can't find /dist/ directory.  Are you running from the Test Pilot root?"
+    exit 1
+fi
+
+if [ -z "$BUCKET" ]; then
+    echo "The S3 bucket is not set. Failing."
+    exit 1
+fi
+
+
+# The basic strategy is to sync all the files that need special attention
+# first, and then sync everything else which will get defaults
+
+# This is not pretty, but until S3 supports arbitrary headers, here we are...
+
+# HTML; 10 minute cache
+aws s3 sync \
+  --cache-control "max-age=600" \
+  --content-type "text/html" \
+  --exclude "*" \
+  --include "*.html" \
+  --metadata '{"x-amz-meta-public-key-pins": "max-age=300;pin-sha256='\''WoiWRyIOVNa9ihaBciRSC7XHjliYS9VwUGOIud4PB18='\'';pin-sha256='\''r/mIkG3eEpVdm+u/ko/cwxzOMo1bk4TyHIlByibiA5E='\''",
+               "x-amz-meta-content-security-policy": "default-src '\''self'\''; connect-src '\''self'\'' analysis-output.telemetry.mozilla.org; font-src '\''self'\'' code.cdn.mozilla.net; form-action '\''none'\''; object-src '\''none'\''; script-src '\''self'\'' www.google-analytics.com; style-src '\''self'\'' code.cdn.mozilla.net; report-uri /__cspreport__;",
+               "x-amz-meta-strict-transport-security": "max-age=31536000; includeSubDomains; preload",
+               "x-amz-meta-x-content-type-options": "nosniff",
+               "x-amz-meta-x-frame-options": "DENY",
+               "x-amz-meta-x-xss-protection": "1; mode=block"}' \
```

Comment:
I don't think that any of these headers change between runs of `aws s3 sync`. Might make sense to also stick these into variables?

```bash
--metadata "{$HSTS,  $HPKP, $CSP, $NOSNIFF, $FRAME_OPTIONS, $XSS}"
```

At worst, it reads much better.